scripts/go/sherpa_onnx.go

```diff
@@ -659,15 +659,15 @@ func (s *OfflineStream) GetResult() *OfflineRecognizerResult {
 	result.Emotion = C.GoString(p.emotion)
 	result.Event = C.GoString(p.event)
 	result.Tokens = make([]string, n)
-	tokens := (*[1 << 10]*C.char)(unsafe.Pointer(p.tokens_arr))[:n:n]
+	tokens := unsafe.Slice(p.tokens_arr, n) //(*[1 << 28]*C.char)(unsafe.Pointer(p.tokens_arr))[:n:n]
```

Comment:
```suggestion
	tokens := unsafe.Slice(p.tokens_arr, n)
```

Can you remove unused code?

We don't need to keep
```go
// (*[1 << 28]*C.char)(unsafe.Pointer(p.tokens_arr))[:n:n]
```

Reply:
yes, I can